src/streamlink/plugins/tvp.py

```diff
@@ -169,8 +172,58 @@ def _get_vod(self, vod_id):
         if data.get("HLS"):
             return HLSStream.parse_variant_playlist(self.session, data["HLS"][0]["src"])
 
+    def _get_tvp_info_vod(self):
+        data = self.session.http.get(
+            self.url,
+            schema=validate.Schema(
+                validate.parse_html(),
+                validate.xml_xpath_string(".//script[contains(text(), 'window.__videoData')]"),
```

Comment:
This implicitly gets the node texts of all matches. May be better to add `[1]/text()`.